docs/UserGuide.md

```diff
@@ -3,436 +3,713 @@ layout: page
+
+Refer to the table below for more details.
+
+| **Flag** | **Expected Argument** | **Description**                                | **Requirements**                                                                                | **Case Sensitivity**  |
+|----------|-----------------------|------------------------------------------------|-------------------------------------------------------------------------------------------------|-----------------------|
+| `n/`     | `<NAME>`              | The client's full name                         | Letters, numbers, and spaces only                                                               | ❌                     |
+| `p/`     | `<PHONE>`             | The client's phone number                      | Valid Singapore phone number:<br/> • 8-digit number<br/> • Starts with 8 or 9                   | ❌                     |
+| `e/`     | `<EMAIL>`             | The client's email address                     | Valid email format (`username@domain.com`)                                                      | ❌                     |
+| `a/`     | `<ADDRESS>`           | The client's physical address                  | No specific restrictions                                                                        | ❌                     |
+| `j/`     | `<JOBNAME>`           | The client's job title or profession           | No specific restrictions                                                                        | ❌                     |
+| `i/`     | `<INCOME>`            | The client's annual income                     | Positive number or zero <br/> • Cannot include commas and decimal points<br/> • Must be numeric | ❌                     |
+| `t/`     | `<TIER>`              | The client's assigned tier level               | Predefined tiers:<br/> • Gold, Silver, Bronze, Reject                                           | ✔️                    |
+| `r/`     | `<REMARK>`            | General remarks about the client               | No specific restrictions                                                                        | ❌                     |
+| `ra/`    | `<REMARK TO APPEND>`  | Append information to the existing remark      | No specific restrictions                                                                        | ❌                     |
+| `rn/`    | `<NEW REMARK>`        | Replaces the existing remark with a new remark | No specific restrictions                                                                        | ❌                     |    
 
-### Feature 3: Remove Old Customer {#remove-old-customer}
+> 💡 **Pro Tip:**
+> 
+> Ensure every flag is followed by a valid argument!
+>
+> Providing a flag without an accompanying argument will result in an error and prevent the command from executing properly.
```

Comment:
Nice, but perhaps replcae `every flag` with `every specified flag`.

Reply:
I feel that `every flag` is concise and to the point - adding "specified" might create unnecessary doubt or confusion about whether there are some flags that don't need arguments.